ast/ast.go

```diff
@@ -1448,14 +1452,13 @@ type CreateTable struct {
 //	CREATE SEQUENCE {{if .IfNotExists}}IF NOT EXISTS{{end}} {{.Name | sql}} }} OPTIONS ({{.Options | sqlJoin ","}})
 type CreateSequence struct {
 	// pos = Create
-	// end = Rparen + 1
+	// end = Options.end
 
 	Create token.Pos // position of "CREATE" keyword
-	Rparen token.Pos // position of ")" of OPTIONS clause
 
 	Name        *Ident
 	IfNotExists bool
-	Options     []*SequenceOption // len(Options) > 0
+	Options     *SequenceOptions
```

Comment:
@makenowjust I changed `[]*SequenceOption` to `SequenceOptions` to keep consistent with ChangeStreamOptions and VectorIndexOptions. And this also can be reused by ALTER SEQUENCE.

Reply:
Good 👍